keyboards/puck/readme.md

```diff
@@ -0,0 +1,2 @@
+# Puck
+An ortholinear number pad.
```

Comment:
Could you use our template for readme files here. 

Here's what the 1up60hse looks like

```
A 60% PCB with USB C, RGB underglow, backlighting, hotswappable switches, and a standard ANSI layout.

Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)  
Hardware Supported: 1up60hse 60% PCB.  
Hardware Availability: [1upkeyboards.com](https://www.1upkeyboards.com/shop/controllers/1up-rgb-pcb-hse/)

Make example for this keyboard (after setting up your build environment):

    make 1upkeyboards/1up60hse:default

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

You don't have to put your name on, some people just write "QMK Community" for the maintainers.